code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -1,9 +1,11 @@
 /* Toxins, poisons, venoms */
 
 /datum/reagent/toxin
-	name = "Toxin"
-	id = "toxin"
+	name = "bitterness"
```

Comment:
....Fuck, thank you for pointing this out. Last time I managed to catch that. (This actually breaks a few things)